src/X.Extensions.Logging.Telegram/X.Extensions.Logging.Telegram.csproj

```diff
@@ -1,28 +1,31 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <LangVersion>default</LangVersion>
-        <Authors>Andrew Gubskiy</Authors>
-        <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
         <Title>X.Extensions.Logging.Telegram</Title>
-        <PackageLicenseUrl>https://github.com/ernado-x/X.Extensions.Logging.Telegram/blob/main/LICENSE</PackageLicenseUrl>
-        <PackageIconUrl>https://ru.gravatar.com/userimage/8071071/f2dc08ee7e4016451f64a7ae9cffd110.png?size=200</PackageIconUrl>
-        <PackageProjectUrl>https://github.com/ernado-x/X.Extensions.Logging.Telegram</PackageProjectUrl>
-        <RepositoryUrl>https://github.com/ernado-x/X.Extensions.Logging.Telegram</RepositoryUrl>
-        <PackageTags>telegram, logging, log, logs, tg</PackageTags>
         <Description>Telegram logging provider</Description>
-        <RepositoryType>git</RepositoryType>
-        <Version>1.0.6.1</Version>
-        <PackageVersion>1.0.6.1</PackageVersion>
-        <TargetFrameworks>net6.0;net7.0;netstandard2.0;netstandard2.1</TargetFrameworks>
+
+        <LangVersion>default</LangVersion>
+        <TargetFrameworks>net6.0;net8.0;netstandard2.0;netstandard2.1</TargetFrameworks>
+
+        <PackageTags>telegram logging log logs</PackageTags>
+
+        <Nullable>enable</Nullable>
+
+        <Authors>Andrew Gubskiy</Authors>
+        <Copyright>Andrew Gubskiy © 2024</Copyright>
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="JetBrains.Annotations" Version="2024.2.0" PrivateAssets="All" />
-      <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.0" />
-      <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
-      <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="8.0.0" />
-      <PackageReference Include="Telegram.Bot" Version="19.0.0" />
+        <None Include="..\..\LICENSE.md" Pack="true" PackagePath=""/>
+        <None Include="..\..\README.md" Pack="true" PackagePath=""/>
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="JetBrains.Annotations" Version="2024.2.0" PrivateAssets="All"/>
```

Comment:
Is it possible to get rid of this is the future? In my mind, it's better to keep fewer deps where it's possible

Reply:
This package reference marked as PrivateAssets, so it will not create dependency on this package from NuGet.